roles/preflight/README.md

```diff
@@ -57,96 +57,178 @@ The idea is use one of them.
     export KUBECONFIG=/var/lib/dci-openshift-app-agent/kubeconfig
     ```
 
-- Create file settings.yml in /etc/dci-openshift-app-agent/settings.yml and provide all the information about your certification projects. Let’s consider two standard scenarios here.
+- Create a dci-pipeline configuration file in my-dci-pipeline-test/pipelines/testpmd-container-preflight-pipeline.yml and provide all the information about your certification projects. Let’s consider two standard scenarios here.
```

Comment:
Same, this pipeline name is quite specific and related to a specific deployment, I'd generalize the pipeline name.
Else, I would clarify at the beginning of this section that this is an example where pipeline configs are saved in `my-dci-pipeline-test/pipelines` folder
This applies also to the e2e certification example provided at the end too